docs/api_reference/src/quantities.tex

```diff
@@ -61,10 +61,10 @@
 
 // \ref{qty.expr.temp.types}, types
 
-template<typename T, typename... Ts>
+template<@\exposconceptnc{SymbolicArg}@ T, @\exposconceptnc{SymbolicArg}@... Ts>
```

Comment:
I've added `SymbolicArg` mostly to verify our implementation that no `const` leaks into the `type_list`. I am not sure if it has to be provided in the wording.